Makefile

```diff
@@ -1,7 +1,7 @@
 PYTHON ?= python3
 
 .PHONY: all
-all: style-check type-check test
+all: format-code style-check type-check test
```

Comment:
```suggestion
all: format style-check type-check test
```

Reply:
i'd also be in favour of changing 'style-check' to 'lint', but these are tiny nits